modules/yardstick/README.txt

```diff
@@ -2,46 +2,93 @@ Yardstick Ignite Benchmarks
 ===========================
 Yardstick Ignite is a set of Ignite Grid (http://ignite.apache.org/) benchmarks written on top of Yardstick framework.
 
-Yardstick Framework
-===================
-Visit Yardstick Repository (https://github.com/gridgain/yardstick) for detailed information on how to run Yardstick benchmarks and how to generate graphs.
 
-The documentation below describes configuration parameters in addition to standard Yardstick parameters.
+Building the project
+====================
+Please look for the building instructions in `DEVNOTES.txt`
+
+
+Running Ignite Benchmarks
+=========================
+
+The simplest way to run benchmarks is running following command in `benchmarks` directory:
```

Comment:
As I mentioned above, we need to generate unique README.txt on the fly for the sources added to the binary and add this section there. "modules/yardstick/README.txt" has to contain only those instructions that are needed if someone wants to build the benchmarks from "modules/yardstick". However, somewhere in this file we are free to refer to Apache binary releases that already include the benchmarks ready to be run.